frontend/src/components/Card.tsx

```diff
@@ -46,9 +46,9 @@ const Card = ({
   }
 
   return (
-    <div className="flex h-fit w-full flex-col items-start justify-normal gap-4 rounded-md border border-border py-6 pl-6 pt-0 md:max-w-6xl md:gap-2">
+    <div className="flex h-fit w-full flex-col items-start gap-4 rounded-md border border-border py-6 pl-4 pt-0 sm:pl-6 md:max-w-6xl md:gap-2">
       <div className="flex w-full flex-wrap items-center justify-between gap-2">
-        <div className="mt-4 flex items-center justify-center gap-2">
+        <div className="mt-4 flex flex-col items-start justify-start gap-2 sm:flex-row sm:items-center">
```

Comment:
Can we make The Icon and the Title for the card appear on the same line? 
![Screenshot 2024-12-21 at 6 28 04 PM](https://github.com/user-attachments/assets/1caa50d9-1020-4fe4-a822-d259e9dc016f)